content/en/apps/guides/database/muting_in_dashboards.md

```diff
@@ -0,0 +1,78 @@
+When a contact gets muted, two of [many things](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/transitions/#muting) happen:
+
+- The target contact and all of its descendants have a `muted` property set equal to the date they were muted
+- an entry is added to the contact's `muting_history` in sentinel's info docs
+
+When building dashboards on KF, you might need to exclude these muted contacts from analytics. An easy way to do this is to check the contact's `muted` property which when present has the date value of when the contact was muted and when absent means that the contact is not muted. This works work when you're only interested in the seeing the latest data but it gets a little bit complicated when you want to look at a contact's mute state from a certain period in the past. 
```

Comment:
Use full words for unfamiliar terms i.e. Klipfolio instead of KF. You could also make it a bit more general and refer to any dashboard platforms. For example, 'When building dashboards on Klipfolio or Superset...'.